service/dap/daptest/gen/main.go

```diff
@@ -0,0 +1,99 @@
+// Binary gen generates service/dap/daptest/responses.go.
+package main
+
+import (
+	"bytes"
+	"flag"
+	"fmt"
+	"go/format"
+	"go/types"
+	"io/ioutil"
+	"os"
+	"text/template"
+
+	"golang.org/x/tools/go/packages"
+
+	_ "github.com/google/go-dap"
+)
+
+var oFlag = flag.String("o", "", "output file name")
+
+var tmpl = template.Must(template.New("assert").Parse(`package daptest
+
+// Code generated by go generate; DO NOT EDIT.
+// The code generator program is in ./gen directory.
```

Comment:
Do you think perhaps this and the output library should be something shipped with go-dap from that repo?